### PR TITLE
Test logic parser

### LINKED ISSUE #104: [Test] Update Logic Parser Testing

### PR DESCRIPTION
**Pull request by ghost**
Test logic parser - should pass all test case

### CODE REVIEW

**Comment by aweijun** [comment]
Good job!
Hi! what do you mean should pass all the test case? 
Does it mean that it has already passed the test case or are you waiting for certain dependencies.

**Comment by lowkaiwei98** [comment]
> Good job! Hi! what do you mean should pass all the test case? Does it mean that it has already passed the test case or are you waiting for certain dependencies.

In this current version, it passed all the test case

**Comment by aweijun** [comment]
Thanks for clarifying!

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -1,140 +1,110 @@
...
+import static seedu.address.logic.commands.CommandTestUtil.STATUS_DESC_COMPLETED;
+import static seedu.address.logic.commands.CommandTestUtil.STATUS_DESC_WATCHING;
+import static seedu.address.logic.commands.CommandTestUtil.TAG_DESC_FRIENDS;
+import static seedu.address.logic.commands.CommandTestUtil.TAG_DESC_MOVIE;
+import static seedu.address.logic.commands.CommandTestUtil.TAG_DESC_OWESMONEY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_ALICE_IN_WONDERLAND;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_STATUS_COMPLETED;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_FRIENDS;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_MOVIE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_OWESMONEY;
```
Comment (aweijun) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:20]: Hi does this fit with the theme of trackermon?
Might be a bit nitpicky but maybe you could change it to something that suits trackermon more instead?
Possible tags could be "HBO"' or "Netflix" etc

**Review by aweijun — COMMENTED** [review]
Other than that minors suggestion, everything else looks good to me!

**Review by JonathanHoshi — APPROVED** [review]
LGTM!

**Review by Ardentsoul — APPROVED** [review]
LGTM!

**Pull request merged**